cluster-support-bot.py

```diff
@@ -5,23 +5,16 @@
 
 import hydra
 import slack
-from slackeventsapi import SlackEventAdapter
 
 
 logging.basicConfig()
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.DEBUG)
 
 
-recent_events = set()  # cache recent event timestamps
-
-# Our app's Slack Event Adapter for receiving actions via the Events API
-slack_signing_secret = os.environ["SLACK_SIGNING_SECRET"]
-slack_events_adapter = SlackEventAdapter(slack_signing_secret, "/slack/events")
+bot_user = os.environ["BOT_USER"]
```

Comment:
`install.md` needs to be updated for these env var changes.

Reply:
done